plugins/node/opentelemetry-instrumentation-graphql/README.md

```diff
@@ -38,12 +40,22 @@ registerInstrumentations({
 
 ```
 
+## Optional Parameters
+
+|    Param    |   type  | Default Value |                                                                        Description                                                                        |   |
+|:-----------:|:-------:|:-------------:|:---------------------------------------------------------------------------------------------------------------------------------------------------------:|:-:|
+|  mergeItems | boolean |     false     |                    Whether to merge list items into a single element. example: `users.*.name` instead of `users.0.name`, `users.1.name`                   |   |
+|    depth    |  number |       -1      |                       The maximum depth of fields/resolvers to instrument. When set to 9 it will not instrument fields and resolvers                      |   |
```

Comment:
```suggestion
|    depth    |  number |       -1      |                       The maximum depth of fields/resolvers to instrument. When set to 0 it will not instrument fields and resolvers                      |   |
```
it would be nice to write down what happens when using the default value of -1

Reply:
added comment about -1